src/styles/Book.css

```diff
@@ -0,0 +1,98 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;900&family=Montserrat&family=Roboto+Slab&display=swap');
+
+:root {
+  --azure: #0290ff;
+  --black-two: #121212;
+}
+
+.book-container {
+  width: 90%;
+  padding: 1.5rem 1rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: #fff;
+  margin: 1.25rem auto;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  border-radius: 4px;
+  justify-content: space-between;
+}
+
+.book-container ul {
+  width: 45%;
+  font-family: 'Roboto Slab', sans-serif;
+  font-weight: bold;
+  display: flex;
```

Comment:
- [ ] Kindly consider using reusable classes for rrepeating CSS rules. For example `display: flex;` has been reused many times in this file, consider creating a reusable CSS class and using that class on the affected elements